### DISCUSSION TITLE
MueLu: Navier Stokes tests

### DISCUSSION THREAD

**Discussion by jhux2** [trilinos/Trilinos#12176] [Solvers] [answered]
@maxfirmbach @mayrmt Does your group utilize the tests in `packages/muelu/test/navierstokes`?  Would it be helpful if these tests were converted to use Tpetra?  There is some ongoing work around the removal of Epetra from active packages, and this set of tests is one of the areas that needs work.

@trilinos/muelu

**Comment by maxfirmbach** [comment, marked as answer]
@jhux2 Sorry for the late reply, currently we are not using and/or relying on these tests :+1: .
